.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "external/debug_assert"]
 	path = external/debug_assert
-	url = https://github.com/foonathan/debug_assert
+url=https://github.com/foonathan/debug_assert/tree/4de9db29ad70faf37bd127edecafa2a12d87999e
```

Comment:
This is wrong, please revert it and use `git submodule update --remote` to update the submodule to the latest commit.